packages/smooth_app/lib/pages/history_page.dart

```diff
@@ -29,6 +30,16 @@ class _HistoryPageState extends State<HistoryPage> {
 
   @override
   Widget build(BuildContext context) {
-    return ProductListPage(productList);
+    final AppLocalizations appLocalizations = AppLocalizations.of(context)!;
+    return Column(
+      children: <Widget>[
+        AppBar(
+          backgroundColor: Colors.white,
+          foregroundColor: Colors.black,
+          title: Text(appLocalizations.history),
+        ),
+        Expanded(child: ProductListWidget(productList)),
+      ],
+    );
```

Comment:
The idea was still to use `ProductListPage`, that would mainly use the new class `ProductListWidget` where most of the page's code would have been moved.

The way the project evolved, `ProductListPage` is now only used by the history page. If you don't use it in the history page anymore, everything you do in `ProductListPage` is basically maintaining dead code.

And I'm not convinced that using a `Column` with an `AppBar` then a body (and without a navigation bar) is such a good idea compared to a standard `Scaffold`.

My suggestion: let history page use `ProductListPage`, that uses the new class `ProductListWidget`.

Reply:
I'd be happy to remove `ProductListPage` if it's unused, but see 
![image](https://user-images.githubusercontent.com/7541/143832676-c0a033b4-bc39-413d-ad7c-385d73b4d3db.png)

I don't remember why I did it using `Column` containing `AppBar`... I changed it to a `Scaffold` now.

Nesting 'pages' seems strange - if we want to use `ProductListPage` only or the history page, then we could just rename it to `HistoryPage`.

Also, see the image above - `ProductList` is still used for the custom product lists (which I don't know if they are going away or not).